assistant_dialog_skill_analysis/inferencing/multi_thread_inference.py

```diff
@@ -58,7 +58,7 @@ def thread_inference(self):
                         break
                     attempt += 1
                     try:
-                        response = inferencer.retrieve_classifier_response(
+                        response = inferencer.skills_util.retrieve_classifier_response(
```

Comment:
Why do you need to access the `inferencer.skills_util.retrieve_classifier_response` can you not call `skills_util.retrieve_classifier_response` and do away with the import